platform/security/handler/security-handler-pki/src/test/java/org/codice/ddf/security/handler/pki/PKIHandlerTest.java

```diff
@@ -138,6 +138,32 @@ public void testGetNormalizedTokenFailureNoCerts() throws ServletException {
     }
 
     /**
+     * Tests that the PKIHandler trows a ServletException when the cert fails to pass the CRL check
+     */
```

Comment:
You may be able to simplify your tests a little if you inject a mock `CrlChecker` instead of using the real one.